cmd/auth/Dockerfile

```diff
@@ -34,6 +34,7 @@ RUN go build \
     -ldflags "-X github.com/vardius/go-api-boilerplate/internal/buildinfo.Version=$VERSION -X github.com/vardius/go-api-boilerplate/internal/buildinfo.GitCommit=$GIT_COMMIT -X 'github.com/vardius/go-api-boilerplate/internal/buildinfo.BuildTime=$(date -u '+%Y-%m-%d %H:%M:%S')'" \
     -a -o /go/bin/app ./cmd/"$BIN"
 
-FROM scratch
+FROM alpine
```

Comment:
we have to figure out which solution works, copying migrations in the init container image or the service image here. maybe actually it is better to keep it in the init container ? and remove it from here ?

Need to know which of them works and consider pros and coms for each option. Let me know your thoughts.

Reply:
Yeah I forgot alpine in it by mistake, let's keep scratch. Actually I removed the migrations from the init container and they still work for the auth service. Seems like the issue was due to the latest tag it was not getting the image of migrate from local repository. This is now solved by replacing latest with local tag.

Reply:
yep in that case what we should do is:

- [ ] remove migrations from service images completely
- [ ] move `migrations` directory into `cmd/migration` service

and then copy migrations into migrations image

Reply:
Fully agree on that but I think there is more to it as well. The migrations by all means need to be completed before the auth and user pods are ready. This in turn can allow us to move the migrations into a completely separate pod and use pod priority. That way there would be no concurrency between the depending pods on the migrations. 
[Example: pod priority](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/)
I will give that a try and add it to this PR if it goes well. Basically the init containers when they fail are very hard to interrupt and debug since they are short-lived by nature. Hopefully by extracting them in a separate pod with highest priority it should become easier to maintain and debug in case of failure. What do you think ?

Reply:
I merged this PR, basic setup is working now major bugs fixed,
I've split namespace and app charts deployment, this way namespaces exists while deploying app into it. Migrations are working as well, however user service grpc adapter fails.

Can you please try and do this pod priority ? I will have a look at user service tomorrow.